cmd/access/node_builder/access_node_builder.go

```diff
@@ -293,7 +293,7 @@ func DefaultAccessNodeConfig() *AccessNodeConfig {
 		versionControlEnabled:                true,
 		storeTxResultErrorMessages:           false,
 		stopControlEnabled:                   false,
-		registerDBPruneThreshold:             pruner.DefaultThreshold,
+		registerDBPruneThreshold:             0,
```

Comment:
Maybe making a name for the value would be more clear?
```suggestion
		registerDBPruneThreshold:             pruner.ThresholdDisabled,
```